components/BirthdayList.js

```diff
@@ -0,0 +1,38 @@
+import { listOfBirthdays } from "../public/lib/birthdays";
+import styled from "styled-components";
+
+export default function BirthdayList() {
+  const birthdays = listOfBirthdays.map((birthday) => {
```

Comment:
I recommend to convert you whole table to a component, to get a nice structure in your code.